src/assets/locales/es.json

```diff
@@ -173,288 +173,288 @@
-  "link.connect_headset": "Ingresar en el dispositivo",
+  "link.in_your_browser": "En el navegador de tu dispositivo, anda a:",
+  "link.enter_code": "Luego, inserta este código de letras desechable:",
+  "link.do_not_close": "Tu cuenta y tu avatar serán transferidos al dispositivo.\nMantén esta página abierta para usar este código.",
+  "link.connect_headset": "Entrar con el dispositivo",
   "link.cancel": "cancelar",
-  "invite.enter_via": "O usa un código temporal ",
-  "invite.enter_via_modal": "Otros pueden ingresar a través de ",
-  "invite.tweet": "tweet",
-  "invite.and_enter_code": " código:",
-  "invite.duration_of_code": "código nuevo cada 72 horas",
-  "invite.or_visit": "Compartir enlace de sala:",
-  "invite.or_visit_modal": "o visitando el enlace permanente",
-  "invite.embed": "o incrustar en una página",
-  "invite.embed-tip": "Ten en cuenta dónde incrustas una sala de %app-name%.\nVe a Configuración de la sala para bloquear los permisos de esta sala.",
-  "commands.fly": "Alternar el modo de vuelo.",
-  "commands.grow": "Aumenta el tamaño de tu avatar.",
-  "commands.shrink": "Disminuye el tamaño de tu avatar.",
+  "invite.enter_via": "O usa el código ",
+  "invite.enter_via_modal": "Otros pueden entrar a través de ",
+  "invite.tweet": "tuit",
+  "invite.and_enter_code": " caducable:",
+  "invite.duration_of_code": "nuevo código cada 72 horas",
+  "invite.or_visit": "Compartir el vínculo de la sala:",
+  "invite.or_visit_modal": "o visitando el vínculo permanente",
+  "invite.embed": "o insértalo en una página",
+  "invite.embed-tip": "Por favor, sé consciente sobre dónde insertas la sala de %app-name%.\nAnda a la configuración de la sala para bloquear los permisos de esta sala.",
+  "commands.fly": "Alternar modo de vuelo.",
+  "commands.grow": "Aumentar el tamaño de tu avatar.",
+  "commands.shrink": "Reducir el tamaño de tu avatar.",
   "commands.scene": "Cambiar la escena.",
-  "commands.rename": "Cambiar el nombre de la habitación.",
-  "commands.help": "Mostrar ayuda.",
-  "commands.leave": "Desconectar de la habitación.",
-  "commands.duck": "El pato funcionó bien. Quack.",
-  "commands.capture": "Captura un video de 15 segundos.",
-  "commands.debug": "Alternar renderizado de depuración física.",
-  "commands.vrstats": "Alternar estadísticas en realidad virtual",
-  "commands.audiomode": "(experimental) Alternar audio posicional.",
-  "preferences.closeButton": "Cerrar el menú de preferencias ",
+  "commands.rename": "Renombrar la sala.",
+  "commands.help": "Mostrar la ayuda.",
+  "commands.leave": "Desconectarse de la sala.",
+  "commands.duck": "El pato pasó bien el test. Cuac.",
+  "commands.capture": "Grabar un video de 15 segundos.",
+  "commands.debug": "Alternar el procesamiento de depuración de la física.",
+  "commands.vrstats": "Alternar estadísticas en VR.",
+  "commands.audiomode": "(experimental) Alternar sonido posicional.",
```

Comment:
"(experimental) Alternar sonido posicional." should be "(experimental) Alternar audio posicional."